README.md

```diff
@@ -113,6 +113,16 @@ Perhaps Goby should be far easier for Rubyists to comprehend. You can use Ruby's
 - Flow control
     - `if`, `else`, `elsif`
     - `while`
+- Literals
+    - Numeric literal
+        - Integer: `100`, `-299`, `-0`
+        - Float: `3.14`, `-273.15`
+    - String literal: `"string"`, `'string'`, `:string`
+    - Symbol literal: `:email`, `{ tel: '9909-999-999' }
```

Comment:
I think `{ tel: '9909-999-999' }` is just a key value representation? it doesn't tell if `tel` a symbol or not. And you missed a '`' quote